docs/backup/08-02-restore-backup.md

```diff
@@ -7,25 +7,56 @@ Follow this tutorial to restore a backed up Kyma cluster. Start with restoring C
 
 ## Prerequisites
 
-To use the restore functionality, download and install the [Velero CLI](https://github.com/heptio/velero/releases/tag/v1.2.0).
+To use the restore functionality, download and install the [Velero CLI](https://github.com/heptio/velero/releases/tag/v1.2.0) based on the `appVersion` in [Chart.yaml](https://github.com/kyma-project/kyma/tree/master/resources/backup/Chart.yaml).
 
 ## Steps
 
 Follow these steps to restore resources:
 
 1. Install the Velero server. Use the same bucket as for backups:
 
-    ```bash
-    velero install \
-        --bucket {BUCKET} \
-        --provider {CLOUD_PROVIDER} \
-        --secret-file {CREDENTIALS_FILE} \
-        --plugins velero/velero-plugin-for-gcp:v1.0.0,eu.gcr.io/kyma-project/backup-plugins:c08e6274 \
-        --restore-only \
-        --wait
-    ```
-
-    >**NOTE**: Check out this [guide](https://velero.io/docs/v1.2.0/customize-installation/) to correctly fill the parameters of this command corresponding to the cloud provider in use.
+    <div tabs name="override-configuration">
+      <details>
+      <summary label="google-cloud-platform">
+      Google Cloud Platform
+      </summary>
+
+      ```bash
+      velero install \
+          --provider gcp \
+          --bucket {BUCKET} \
+          --secret-file {CREDENTIALS_FILE} \
+          --plugins velero/velero-plugin-for-gcp:v1.0.0,eu.gcr.io/kyma-project/backup-plugins:c08e6274 \
+          --restore-only \
+          --wait
+      ```
+
+      >**NOTE:** For details on configuring and installing Velero on GCP, see [this](https://github.com/vmware-tanzu/velero-plugin-for-gcp) repo.
```

Comment:
```suggestion
      >**NOTE:** For details on configuring and installing Velero on GCP, see [this](https://github.com/vmware-tanzu/velero-plugin-for-gcp) repository.
```
We use the full term names (https://kyma-project.io/community/guidelines/content/#style-and-terminology-style-and-terminology-terminology) --> so `repository` instead of `repo`.